sdk/search/Azure.Search.Documents/tests/Samples/Sample05_IndexingDocuments.cs

```diff
@@ -71,7 +71,7 @@ public IEnumerable<Product> GenerateCatalog(int count = 1000)
         }
         #endregion
 
-        private async Task<SearchClient> CreateIndexAsync(SearchResources resources)
+        private async Task<SearchClient> CreateIndexAsync(SearchResources resources, bool instrumentClient = true)
```

Comment:
I would recommend instrumenting and using https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.TestFramework/src/ClientTestBase.cs#L115 where required.

Reply:
Nice. I have switched to using that.